scrmbl/scrmbl.py

```diff
@@ -19,3 +20,4 @@ def echo(message: str, charset: List[str] = charsets.ALL, speed: float = 0.05,
             print('\r{}{}'.format(echoed, character), end='')
             echoed = echoed + character
         print()
+
```

Comment:
[Triggers PEP8 W391](https://travis-ci.org/etienne-napoleone/scrmbl/jobs/437932634#L539)
Please remove the trailing blank line